### PR TITLE
Update A.html

### PR DESCRIPTION
**Pull request by ManarOmari**
first ,update readme file to fit the template 
second , remove a.gitignore file and create .gitignore file and add test.md and other file 
third, rename a.html to index.html 
finally adding description for this pull request

### CODE REVIEW

**Review by Mre55 — COMMENTED** [review]
**Hi @ManarOmari,**

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :negative_squared_cross_mark:

You have submitted a project, that some linter check is not passing, don't have a pull request description which shows a short summary of what you did in this pull request. 

- [x] Kindly use [this](https://github.com/microverseinc/readme-template/blob/master/README.md) REAMDE template and edit it to your own version.
- [x] Please edit your `a.gitignore` file name to be `.gitignore` which is the proper gitignore file.
- [x] And add `test.md` file and `node_module` to `.gitignore` file.
- [x] Kindly add `.html` and `.css` file with separate commits you can do it by deleting what you have now and add them in separate commit. 



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Comment by ManarOmari** [comment]
Dear ,
I have spent 3 hours to understand GitHub and how to add files to .gitignore
And I think it was working fine!!

I will try to update this now .
Thank you

**Comment by ManarOmari** [comment]
Dear ,
can you help me ?
should I create new repo and do all things again ?

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @ManarOmari, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly add a description to your pull request. Descriptions explain the changes introduced on a branch Click the three-dotted lines in the image below
image

- [x]  Kindly follow the given template for the readme file. download the readme file template from [this link](https://github.com/microverseinc/readme-templatel) and replace the one you currently have with it. Also, fill out the project name and all the other necessary information such as linked in and github usernames and profile links.

Your readme
image

given readme
image

- [x] Kindly change the file name in the image below to `.gitignore`
image

- [x]  Add `test.md` file and `node_modules` to .gitignore file.

- [x]  Add linters, refer to these [instructions](https://github.com/microverseinc/linters-config/tree/master/html-css). Your linters should pass and be green.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ManarOmari** [comment]
I updated all the issues, please look to readMe file

**Comment by ManarOmari** [comment]
first ,update readme file to fit the template
second , remove a.gitignore file and create .gitignore file and add test.md and other file
third, rename a.html to index.html
finally adding description for this pull request

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
-!.gitignore
+#.gitignore
```
Comment (Ismailco) [inline .gitignore:1]: - [ ] [Optional] This Pull Request should have a descriptive title telling you what changes are inside, you can change it so it reflects the content of the PR

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
 node_modules/
 test.md
```
Comment (Ismailco) [inline .gitignore:3]: - [ ] Good job adding the ```test.md``` file to the .gitignore, however, you need to remove it from your branch. you can remove ```test.md``` using [this tutorial](https://fedingo.com/how-to-remove-git-ignore-files-from-git-repository/)

File: .gitignore (hunk 1 above)
Comment (Ismailco) [inline .gitignore:1]: - [ ] [Optional] This is optional but you can remove unused headers from your README file so it looks more professional,

Screenshot_20220526_175356

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @ManarOmari,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @ManarOmari , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**